modules/assim.batch/R/pda.define.llik.R

```diff
@@ -0,0 +1,34 @@
+##' Define PDA Likelihood Functions
+##'
+##' @title Define PDA Likelihood Functions
+##' @param all params are the identically named variables in pda.mcmc / pda.emulator
+##'
+##' @return List of likelihood functions, one for each dataset to be assimilated against.
+##'
+##' @author Ryan Kelly
+##' @export
+pda.define.llik.fn <- function(settings) {
+  # Currently just returns a single likelihood, assuming the data are flux NEE.
+  llik.fn <- list()
+  for(i in 1:length(settings$assim.batch$inputs)) {
+    # NEE + heteroskedastic Laplace likelihood
+    if(settings$assim.batch$inputs[[i]]$variable.id == 297 && 
```

Comment:
This shouldn't depend on variable.id, just the likelihood type. Provided the parameters are different, the same function should work for other variables (e.g. LE)